transcriptomics_data_service/db.py

```diff
@@ -135,6 +135,16 @@ async def fetch_gene_expressions_by_experiment_id(self, experiment_result_id: st
             res = await conn.fetch(query, experiment_result_id)
         return tuple([self._deserialize_gene_expression(record) for record in res])
 
+    async def fetch_gene_expressions(
```

Comment:
Do you see a use for this function in the future?
The way it is being used now is identical to `fetch_gene_expressions_by_experiment_id`, so I would remove it for now and implement pagination in a separate PR.

Reply:
You are rigth, it is removed and reimplemented in PR#15